src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -78,6 +78,10 @@
                                 (filter (fn [[k _]]
                                           (.startsWith (name k) "library:")))
                                 (into {}))
+        tags (->> attributes
+                  (filter (fn [[k _]]
+                            (.startsWith (name k) "tag:")))
+                  (into {}))
```

Comment:
This whole thing could just be `(utils/filter-keys #(.startsWith (name %) "tag:") attributes)`